src/com/hotmail/kalebmarc/textfighter/main/Ui.java

```diff
@@ -112,6 +112,16 @@ public static void println() {
         print("\n");
     }
 
+    private static int hrLength = 66;
+    public static void printhr() {
+        println("-".repeat(hrLength));

```

Comment:
`.repeat` requires java 11+, which is fine, but could you add a note in the README stating java 11+ is required? :) It's common for java developers to use much older versions like java 8, 9, etc